java/src/main/java/com/adhoc/flight/client/AdhocFlightClient.java

```diff
@@ -201,6 +203,8 @@ public FlightStream getStream(FlightInfo flightInfo, CallOption... options) {
      * @param headerCallOption client properties to execute provided SQL query with.
      * @param rowProcessor object to use for processing rows
      * @throws Exception if error occurs during query execution.
+     * @deprecated
```

Comment:
Can we just delete functions/code that are not being used?

Reply:
Solved in the new version.